pom.xml

```diff
@@ -35,7 +35,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <authlete-java-common.version>2.97</authlete-java-common.version>
+        <authlete-java-common.version>2.82</authlete-java-common.version>
```

Comment:
Why was the java-common version downgraded? Current version is `2.99` as of today.